managed-services/iot-hub/reference-content/certificate.mdx

```diff
@@ -0,0 +1,67 @@
+---
+meta:
+  title: Updating your Endpoint's Certificate Authority 
+  description: This page provides important information about the Scaleway IoT Hub Certificate Authority.
+content:
+  h1: Certificate Authority update
+  paragraph: This page provides important information about the Scaleway IoT Hub Certificate Authority.
+dates:
+  validation: 2023-04-27
+  posted: 2023-04-27
+categories:
+  - managed-services
+tags: iot iot-hub ca certificate authority update
+---
+
+As a security measure, Scaleway IoT Hub was designed with a mutual Transport Layer Security (TLS) as the default authentication protocol. This means each device needs both a certificate and key pair to connect to IoT Hub.
```

Comment:
> This means each device needs both a certificate and key pair to connect to IoT Hub.

This is not the important part here, let's concentrate on the certificate concerned by this migration ;)
So something like:

> This means that each device need to verify the authenticity of the IoT Hub certificate.

Then the next paragraph explains the CA, so this should be enough. ;)